### PR TITLE
[MM-21000] Fix using local storage to return to a DM/GM

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary
Fix whenever using local storage to return to a previous DM/GM as we were only looking for channels.

#### Ticket Link

[MM-21000](https://mattermost.atlassian.net/browse/MM-21000)

### CODE REVIEW

File: actions/global_actions.jsx [hunk 1]
```diff
@@ -14,8 +14,9 @@ import {logout, loadMe} from 'mattermost-redux/actions/users';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import {getCurrentTeamId, getMyTeams, getTeam, getMyTeamMember, getTeamMemberships} from 'mattermost-redux/selectors/entities/teams';
 import {getCurrentUser, getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
-import {getCurrentChannelStats, getCurrentChannelId, getMyChannelMember, getRedirectChannelNameForTeam, getChannelsNameMapInTeam} from 'mattermost-redux/selectors/entities/channels';
+import {getCurrentChannelStats, getCurrentChannelId, getMyChannelMember, getRedirectChannelNameForTeam, getChannelsNameMapInTeam, getAllDirectChannels} from 'mattermost-redux/selectors/entities/channels';
 import {ChannelTypes} from 'mattermost-redux/action_types';
+import {getGroupsAssociatedToTeam} from 'mattermost-redux/selectors/entities/groups';
```
Comment (jespino) [inline actions/global_actions.jsx]: This are not group messages, this are ldap groups you have to look for a selector in the selectors/channels.ts

Comment (Willyfrog) [reply actions/global_actions.jsx]: oh! then it's not needed, thanks!

**Comment by hmhealey** [comment]
Is there any way we could add tests for this, either unit tests or E2E ones?

**Comment by Willyfrog** [comment]
@hmhealey added, thanks for the suggestion :)

**Pull request merged**